x/leverage/simulation/operations_test.go

```diff
@@ -29,15 +29,32 @@ type SimTestSuite struct {
 // SetupTest creates a new umee base app
 func (s *SimTestSuite) SetupTest() {
 	checkTx := false
-	s.app = umeeappbeta.Setup(s.T(), checkTx, 1)
-	s.ctx = s.app.BaseApp.NewContext(checkTx, tmproto.Header{
-		ChainID: fmt.Sprintf("test-chain-%s", tmrand.Str(4)),
-		Height:  1,
-	})
+	betaApp := umeeappbeta.Setup(s.T(), checkTx, 1)
+	ctx := betaApp.NewContext(checkTx, tmproto.Header{})
+
+	// Note: Setting umee collateral weight to 1.0 to allow lender to borrow heavily
+	umeeToken := types.Token{
+		BaseDenom:            umeeapp.BondDenom,
+		ReserveFactor:        sdk.MustNewDecFromStr("0.25"),
+		CollateralWeight:     sdk.MustNewDecFromStr("1.0"),
+		BaseBorrowRate:       sdk.MustNewDecFromStr("0.02"),
+		KinkBorrowRate:       sdk.MustNewDecFromStr("0.2"),
+		MaxBorrowRate:        sdk.MustNewDecFromStr("1.0"),
+		KinkUtilizationRate:  sdk.MustNewDecFromStr("0.8"),
+		LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+	}
+
+	betaApp.LeverageKeeper.SetRegisteredToken(ctx, umeeToken)
+	betaApp.LeverageKeeper.SetExchangeRate(ctx, umeeapp.BondDenom, sdk.MustNewDecFromStr("0.9"))
+	betaApp.OracleKeeper.SetExchangeRate(ctx, umeeapp.DisplayDenom, sdk.OneDec())
+	leverage.InitGenesis(ctx, betaApp.LeverageKeeper, *types.DefaultGenesis())
+
+	s.app = betaApp
+	s.ctx = ctx
 }
 
-// getTestingAccounts generates
-func (s *SimTestSuite) getTestingAccounts(r *rand.Rand, n int) []simtypes.Account {
+// getTestingAccounts generates accounts with stake and umee balance
+func (s *SimTestSuite) getTestingAccounts(r *rand.Rand, n int, cb func(fundedAccount simtypes.Account)) []simtypes.Account {
 	accounts := simtypes.RandomAccounts(r, n)
 
```

Comment:
After adding multiple accepted assets, also be sure to mint some of every type to the accounts.

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/fe4fa741d42ca44780be0911235763b174e3bf4f